meson.build

```diff
@@ -2896,14 +2904,35 @@ meson_install_args = meson_args + ['install'] + {
     'muon': []
 }[meson_impl]
 
+# since install_test_files are depending tmp_install
+# be sure about installing test files later
+tmp_install_priority = 100
```

Comment:
Is this still needed? I guess it's still worth avoiding meson install and this doing the same thing concurrently. But then the comment needs to be rephrased.

Reply:
I wanted to avoid meson install and this doing the same thing concurrently but instead of doing this, does adding "suite: ['setup']" to install_additional_files makes more sense?

Reply:
Yes, it should be part of the setup suite.